src/core/lib/event_engine/posix_engine/tcp_socket_utils.cc

```diff
@@ -125,7 +125,7 @@ absl::Status PrepareTcpClientSocket(PosixSocketWrapper sock,
   return absl::OkStatus();
 }
 
-#endif  // GRPC_POSIX_SOCKET_UTILS_COMMON
+#endif /* GRPC_POSIX_SOCKET_UTILS_COMMON */
```

Comment:
Why change these? This undoes some recent work in converting c-style comments to c++-style comments. https://github.com/grpc/grpc/pull/31923

Reply:
Fixed.